src/views/statutory/EditCandidate.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="tile is-ancestor ">
     <div class="tile is-child">
+      <span class="notification is-info" v-if="$route.params.candidate_id">
+        To redraw your candidature, you should send an email to the Juridical Commission via <u>jc@aegee.eu</u>.
```

Comment:
```suggestion
        To withdraw your candidature, you should send an email to the Juridical Commission via <u>jc@aegee.eu</u>.
```

Reply:
And maybe a mailto:

Reply:
Hahah yess that is what I wanted to type, oops